PLUB/Sources/Views/Home/BookmarkViewController.swift

```diff
@@ -0,0 +1,237 @@
+        owner.interesetListGridButton.isSelected = false
+        owner.delegate?.didTappedInterestListChartButton()
+      })
+      .disposed(by: disposeBag)
+    
+    interestListChartButton.buttonUnTapObservable
+      .withUnretained(self)
+      .subscribe(onNext: { owner, _ in
+        owner.interestListChartButton.isSelected = true
+      })
+      .disposed(by: disposeBag)
+    
+    interesetListGridButton.buttonTapObservable
+      .withUnretained(self)
+      .subscribe(onNext: { owner, _ in
+        owner.interestListChartButton.isSelected = false
+        owner.delegate?.didTappedInterestListGridButton()
+      })
+      .disposed(by: disposeBag)
+    
+    interesetListGridButton.buttonUnTapObservable
+      .withUnretained(self)
+      .subscribe(onNext: { owner, _ in
+        owner.interesetListGridButton.isSelected = true
+      })
+      .disposed(by: disposeBag)
+  }
+}
+
+final class BookmarkViewController: BaseViewController {
+  
+  private let viewModel: BookmarkViewModelType
+  
+  private var model: [SelectedCategoryCollectionViewCellModel] = [] {
+    didSet {
+      interestListCollectionView.reloadData()
+    }
+  }
+  
+  private var selectedCategoryType: SelectedCategoryType = .chart
+  
+  private lazy var headerView = BookmarkHeaderView().then {
+    $0.delegate = self
+  }
+  
+  private lazy var interestListCollectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout().then {
+    $0.scrollDirection = .vertical
+  }).then {
+    $0.backgroundColor = .background
+  }.then {
+    $0.register(SelectedCategoryGridCollectionViewCell.self, forCellWithReuseIdentifier: SelectedCategoryGridCollectionViewCell.identifier)
+    $0.register(SelectedCategoryChartCollectionViewCell.self, forCellWithReuseIdentifier: SelectedCategoryChartCollectionViewCell.identifier)
+    $0.delegate = self
+    $0.dataSource = self
+  }
+  
+  init(viewModel: BookmarkViewModelType = BookmarkViewModel()) {
+    self.viewModel = viewModel
+    super.init(nibName: nil, bundle: nil)
+  }
+  
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override func setupLayouts() {
+    super.setupLayouts()
+    [headerView, interestListCollectionView].forEach { view.addSubview($0) }
+    headerView.snp.makeConstraints {
+      $0.top.equalTo(view.safeAreaLayoutGuide)
+      $0.leading.trailing.equalToSuperview()
+      $0.height.equalTo(32)
+    }
+    
+    interestListCollectionView.snp.makeConstraints {
+      $0.top.equalTo(headerView.snp.bottom)
+      $0.directionalHorizontalEdges.bottom.equalToSuperview()
+    }
+  }
+  
+  override func setupConstraints() {
+    super.setupConstraints()
+  }
+  
+  override func setupStyles() {
+    super.setupStyles()
+    self.navigationItem.title = nil
+    self.navigationItem.leftBarButtonItems = [
+      UIBarButtonItem(image: UIImage(named: "back"), style: .done, target: self, action: #selector(didTappedBackButton)),
```

Comment:
이미지 리소스 이름이 잘못되었습니다. `Back`으로 처리하셔야 보입니다.
```suggestion
      UIBarButtonItem(image: UIImage(named: "Back"), style: .done, target: self, action: #selector(didTappedBackButton)),
```

Reply:
<img width="306" alt="스크린샷 2023-02-17 오후 11 26 31" src="https://user-images.githubusercontent.com/39263235/219681574-6d22bea9-613a-457a-aa8b-99f651d16e53.png">
엇 지금 제 xcode상으로는 back이 맞는거같은데 혹시 제 에셋이 최신화가 안되있는걸까요 ??

Reply:
저는 건준님 pr 그대로 가져다가 테스트를 해본 것 뿐인데 서로 다르게 보이네요.. 흠 이상하네요.

![image](https://user-images.githubusercontent.com/57972338/219819921-e0a4a84f-9722-4b46-9c13-135e547ecafa.png)

Reply:
일단 해당 이미지 삭제 이후 다시 재삽입해놓고 pr올렸습니다 👍👍